packages/contracts/package.json

```diff
@@ -32,7 +32,7 @@
     "deploy:goerli:minter": "yarn hardhat run --network goerli scripts/minter-deployments/V3/generic-minter-deployer-v3core.ts",
     "deploy:mainnet:minter": "yarn hardhat run --network mainnet scripts/minter-deployments/V3/generic-minter-deployer-v3core.ts",
     "pre-commit": "yarn lint",
-    "pre-push": "yarn test:parallel"
+    "pre-push": "echo no pre-push action in contracts package"
```

Comment:
I think this is the right way to do this? because we have a pre-push in one of the other packages as well, so keeping the script, but doing nothing